### ISSUE TITLE
Can I use KSCrash to print live CallStack on local

### ISSUE THREAD

**Issue by zixun** [kstenerud/KSCrash#162]
Can I use KSCrash to print live CallStack on local? such as NSLog()

**Comment by Meonardo** [comment]
好像不行, (当前的在 debug 栏不就可以看到么?)但是能打印上一次崩溃的日志, KSCrash 创建一个实例 直接调用 allReports. 另外可以先运行一遍等应用挂掉之后, 在进行调试也是可以的

**Comment by kstenerud** [comment]
If you want to get the program state ad-hoc, you could call `[KSCrash reportUserException]`. That will generate a report with the current callstack, and you can tell it not to terminate the program.

**Comment by zixun** [comment]
@kstenerud  but how to print CallStack,  [KSCrash reportUserException] returns void

**Comment by kstenerud** [comment]
`reportUserException` only stores the exception. You still have to send reports (like you do when the app first starts up in the example apps). What I'll do if I want to send or print immediately is make the call to `reportUserException` and then immediately call `sendAllReportsWithCompletion`. Using a `KSCrashReportSinkConsole` sink will make it print to the console.

**Comment by naftaly** [comment]
Yes, you can capture live call stacks. See KSBacktrace APIs for capturing the current thread's backtrace.
